release.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Release new version of artman.
+
+This script updates SHA hashes of latest commits for both gapic-generator
+and googleapis, and updates version number for subsequent (auto-)release.
+
+Usage:
+$ python3 release.py googleapis [sha]      # update googleapis commit hash
+$ python3 release.py gapic-generator [sha] # update gapic-generator commit hash
+$ python3 release.py 1.12.4                # update artman version
```

Comment:
Since this script doesn't actually release anything, I'd prefer a different name, like maybe `update_versions.py` or `bump_versions.py` or whatever.

Also, the need to call it multiple times feels a little odd. I would prefer something like the following:

```
# fetch automatically
$ python3 update_versions.py --fetch-googleapis --fetch-gapic-generator 1.12.4`

# manual shas
$ python3 update_versions.py --update-googleapis <sha> --update-gapic-generator <sha> 1.12.4`
```

Reply:
I spent some time thinking about it. Basically, updating gapic-generator commit sha does not force us to release a new artman version immediately, and vice versa, so if we go with named parameters, we'll need one for version as well.  How about this:

```sh
$ python3 update_versions.py --fetch-googleapis
$ python3 update_versions.py --set-googleapis <sha>
$ python3 update_versions.py --fetch-gapic-generator
$ python3 update_versions.py --set-gapic-generator <sha>
$ python3 update_versions.py --set-version <version>
```
(options can be combined in one invocation)

Does it make sense?

Reply:
My thought was that if someone wanted to only update a sha, they could just modify the file without having to go through the complexity of running the command - the value I see for this is the "maintenance" case of "just update everything & prepare for the next release".